### PR TITLE
Plotting large dataset in Dash app

### PR DESCRIPTION
**Pull request by bchen39**
Please also check the links to my test applications, using [WebGL](https://plotly-large-dataset-webgl.ploomberapp.io/), [`plotly-resampler`](https://plotly-large-dataset-resampler.ploomberapp.io/) and the two methods [combined](https://plotly-large-dataset-combined.ploomberapp.io/)

Closes [ploomber.io#199](https://github.com/ploomber/ploomber.io/issues/199)

----
📚 Documentation preview 📚: https://ploomber-doc--267.org.readthedocs.build/en/267/

### CODE REVIEW

File: examples/dash/plotly-large-dataset/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+![](static/app_resampler.png)
+
+- Combining the above approaches (in `combined` folder).
+
+![](static/app_combined.png)
+
+We will be using a commercial flight dataset that documents information such as flight departure date/time and delays. You can find it [here](https://github.com/vega/falcon/blob/master/data/flights-3m.csv).
+
+Once you download the dataset, run `python csv-clean.py flights-3m.csv` to obtained the cleaned csv file `flights-3m-cleaned.csv`. Move the cleaned file to the `data` folder in any of the project folders (`webgl`, `resample` or  `combined`) you want to test.
```
Comment (bryannho) [inline examples/dash/plotly-large-dataset/README.md:18]: `obtained` -> `obtain`

**Review by bryannho — COMMENTED** [review]
looks good, one small comment

**Comment by neelasha23** [comment]
Please add a brief description about the app and how users need to interact with it, what output should they expect to see (somewhere at the top):

Screenshot 2024-08-13 at 10 43 56 AM

**Comment by bchen39** [comment]
> Please add a brief description about the app and how users need to interact with it, what output should they expect to see (somewhere at the top):
> 
> Screenshot 2024-08-13 at 10 43 56 AM
> You can mention that date and time is in 24 hr format.
> 
> Also, is there an easy way to select the date/year and then displaying the corresponding time period's data? Maybe a datepicker? Right now we need to zoom out completely to view all the years.

How about a layout like this?

Screenshot 2024-08-13 at 9 48 50 AM

**Comment by neelasha23** [comment]
Looks fine. Make sure to address the other points as well @bchen39

**Review by bryannho — DISMISSED** [review]
lgtm

**Comment by neelasha23** [comment]
The date filters look fine but there is still no description in the app itself.
 Refer to this part of the above comment:

> Please add a brief description about the app and how users need to interact with it, what output should they expect to see (somewhere at the top):

You can also mention the date range of the dataset. I think it just has data for 2006? @bchen39

**Comment by bchen39** [comment]
> The date filters look fine but there is still no description in the app itself. Refer to this part of the above comment:
> 
> > Please add a brief description about the app and how users need to interact with it, what output should they expect to see (somewhere at the top):
> 
> You can also mention the date range of the dataset. I think it just has data for 2006? @bchen39

Ok. For clarification, by "description" you are referring to the code for `app.py`, the output, and how users can interact with the app. Is that correct?

**Comment by neelasha23** [comment]
No need to mention any code. Just below `Plotting Large Datasets in Dash` you can mention very briefly that dataset contains flights data of X year - Y year (replace with exact values). What the plot represents. And that users can zoom into the plot or change date filters and view the plot update.

**Comment by bchen39** [comment]
Added [date range](https://github.com/ploomber/doc/pull/267/commits/432b981323631e62ba0a79d42e7999f872d4c728#diff-38bc3745e473efe23597e5219dcfbcc463fbaefe688330dd4a57d1c6b6d402c3R10) of our dataset and [description](https://github.com/ploomber/doc/pull/267/commits/432b981323631e62ba0a79d42e7999f872d4c728#diff-38bc3745e473efe23597e5219dcfbcc463fbaefe688330dd4a57d1c6b6d402c3R14-R36) of the app display and how to interact with it.

**Comment by bryannho** [comment]
@bchen39 I think she means to add this description as text on the actual Dash app display, not in the readme. So that users can read it when they visit the demo

**Comment by bchen39** [comment]
Ahh ok. I will do so

**Comment by bchen39** [comment]
Let me know if this display works! @neelasha23 @bryannho 

Screenshot 2024-08-13 at 1 32 29 PM

**Review by neelasha23 — DISMISSED** [review]

**Comment by bchen39** [comment]
My apologies, I forgot to push the modification to the apps. Please check the updated [demo](https://plotly-large-dataset-combined.ploomberapp.io/). Thank you!

**Review by bryannho — APPROVED** [review]

**Pull request merged**